src/olympia/activity/views.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework.mixins import ListModelMixin, RetrieveModelMixin
+from rest_framework.viewsets import GenericViewSet
+
+from olympia import amo
+from olympia.activity.serializers import ActivityLogSerializer
+from olympia.addons.views import AddonChildMixin
+from olympia.api.permissions import (
+    AllowAddonAuthor, AllowReviewer, AllowReviewerUnlisted, AnyOf)
+from olympia.devhub.models import ActivityLog
+from olympia.versions.models import Version
+
+
+class VersionReviewNotesViewSet(AddonChildMixin, RetrieveModelMixin,
+                                ListModelMixin, GenericViewSet):
+    permission_classes = [
+        AnyOf(AllowAddonAuthor, AllowReviewer, AllowReviewerUnlisted),
+    ]
+    serializer_class = ActivityLogSerializer
+    queryset = ActivityLog.objects.all()
+    filter = amo.LOG_REVIEW_QUEUE_DEVELOPER
+
+    def get_queryset(self):
+        self.get_addon_object()  # Just to force the permission checks.
+        version_object = Version.unfiltered.all().get(
+            pk=self.kwargs['version_pk'])
```

Comment:
I had not thought of that problem earlier, but you need a `get_object_or_404` here to avoid 500s if the version does not exist.